gerrychain/proposals/tree_proposals.py

```diff
@@ -1,15 +1,16 @@
 from functools import partial
-from ..random import random
 
+from ..random import random
 from ..tree import (
-    recursive_tree_part, bipartition_tree, bipartition_tree_random,
+    bipartition_tree_random,
     _bipartition_tree_random_all, uniform_spanning_tree,
-    find_balanced_edge_cuts_memoization,
+    find_balanced_edge_cuts_memoization, bipartition_tree_retworkx
+    # recursive_tree_part, bipartition_tree,
 )
 
 
 def recom(
-    partition, pop_col, pop_target, epsilon, node_repeats=1, method=bipartition_tree
+    partition, pop_col, pop_target, epsilon, node_repeats=1, method=bipartition_tree_retworkx
```

Comment:
Reminder to self: the `method` param gets ignored rn -- this should be fixed/changed.